src/inline_snapshot/_utils.py

```diff
@@ -112,6 +112,9 @@ class simple_token(namedtuple("simple_token", "type,string")):
 
     def __eq__(self, other):
         if self.type == other.type == 3:
+            if self.string[0] == "f" or other.string[0] == "f":
```

Comment:
f-strings can also start with `F` or `rf`

Reply:
Yes, you're right, thank you. I'm a bit sick and tired at the moment. I will fix this later. I hope the current fix works for logfire.